well-known-bots.json

```diff
@@ -7503,6 +8208,7 @@
     ],
     "pattern": "Datadog\\/{0,1}Synthetics",
     "addition_date": "2024/09/19",
+    "verification": [],
```

Comment:
I thought we were verifying datadog, no?

Reply:
They only support IP verification, which is not covered by this PR.